www/js/controllers/customer_menu_controller.js

```diff
@@ -0,0 +1,27 @@
+app.controller('CustomerMenuCtrl', function($scope, $state, $ionicLoading, $ionicPopup, UserService, $ionicHistory) {
+
+  $scope.customer = UserService.currentCustomer();
+
+  $scope.showBalance = function() {
+    if ($scope.customer) {
```

Comment:
What is the purpose of this `if` block? Seems that it will make the balance button do nothing if there is no customer. But that is not really an acceptable failure state. If there is no customer than we shouldn't even be on the customer menu at all, and so the app may as well fail loudly, because it has already failed.